src/Microsoft.Build.Sql/Microsoft.Build.Sql.csproj

```diff
@@ -9,16 +9,19 @@
 
     <!-- Path where all the DLLs build depends on will be copied to -->
     <BuildBinariesPath>$(MSBuildThisFileDirectory)\tools\netstandard2.1\</BuildBinariesPath>
+
+    <DacFxPackageVersion Condition="'$(DacFxPackageVersion)' == ''">160.5332.1-preview</DacFxPackageVersion>
```

Comment:
Should (can) this be pushed up to a props file so that this value can be shared between here and the Tests sqlproj?